lib/samples/style_graphics_with_renderer/style_graphics_with_renderer.dart

```diff
@@ -0,0 +1,276 @@
+    );
+    final pointGraphic = Graphic(geometry: pointGeometry);
+    // Add the graphic to the graphics overlay.
+    pointGraphicsOverlay.graphics.add(pointGraphic);
+
+    return pointGraphicsOverlay;
+  }
+
+  GraphicsOverlay getLineGraphicsOverlay() {
+    // Create a simple line symbol.
+    final lineSymbol = SimpleLineSymbol(
+      color: Colors.blue,
+      width: 5,
+    );
+    // Create a graphics overlay for the line.
+    final lineGraphicsOverlay = GraphicsOverlay();
+    // Create and assign a simple render to the graphics overlay.
+    lineGraphicsOverlay.renderer = SimpleRenderer(symbol: lineSymbol);
+
+    // Create a line with `Polyline` geometry.
+    final lineBuilder =
+        PolylineBuilder(spatialReference: SpatialReference.webMercator);
+    lineBuilder.addPointXY(x: -10e5, y: 40e5);
+    lineBuilder.addPointXY(x: 20e5, y: 50e5);
+    final lineGraphic = Graphic(geometry: lineBuilder.toGeometry());
+    // Add the graphic to the graphics overlay.
+    lineGraphicsOverlay.graphics.add(lineGraphic);
+
+    return lineGraphicsOverlay;
+  }
+
+  GraphicsOverlay getEllipseGraphicsOverlay() {
+    // Create a simple fill symbol.
+    final ellipseFillSymbol = SimpleFillSymbol(color: Colors.purple);
+    // Create a graphics overlay for the ellipse.
+    final ellipseGraphicsOverlay = GraphicsOverlay();
+    // Create and assign a simple renderer for the ellipse.
+    ellipseGraphicsOverlay.renderer = SimpleRenderer(symbol: ellipseFillSymbol);
+
+    // Create an ellipse graphic.
+    final ellipseCenter = ArcGISPoint(
+      x: 40e5,
+      y: 25e5,
+      spatialReference: SpatialReference.webMercator,
+    );
+    final parameters = GeodesicEllipseParameters(
+      axisDirection: -45,
+      angularUnit: AngularUnit(unitId: AngularUnitId.degrees),
+      center: ellipseCenter,
+      linearUnit: LinearUnit(unitId: LinearUnitId.kilometers),
+      maxPointCount: 100,
+      maxSegmentLength: 20,
+      geometryType: GeometryType.polygon,
+      semiAxis1Length: 200,
+      semiAxis2Length: 400,
+    );
+    final ellipseGeometry =
+        GeometryEngine.ellipseGeodesic(parameters: parameters);
+    final ellipseGraphic = Graphic(geometry: ellipseGeometry);
+
+    // Add the graphic to the overlay.
+    ellipseGraphicsOverlay.graphics.add(ellipseGraphic);
+    return ellipseGraphicsOverlay;
+  }
+
+  GraphicsOverlay getCurvedPolygonGraphicsOverlay() {
+    // Create a simple fill symbol with outline.
+    final curvedLineSymbol = SimpleLineSymbol(color: Colors.black);
+    final curvedFillSymbol = SimpleFillSymbol(
+      color: Colors.red,
+      outline: curvedLineSymbol,
+    );
+    // Create a graphics overlay for the polygons with curve segments.
+    final curvedGraphicsOverlay = GraphicsOverlay();
+    // Create and assign a simple renderer to the graphics overlay.
+    curvedGraphicsOverlay.renderer = SimpleRenderer(symbol: curvedFillSymbol);
+
+    // Create a heart-shape graphic from ArcGISSegment's.
```

Comment:
ArcGISSegment is just Segment -- we didn't add the ArcGIS prefix to that one.

Reply:
Done.